web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -573,6 +573,16 @@ $.extend(fixmystreet.set_up, {
             selector = "[data-category='" + category + "']";
         $("form#report_inspect_form [data-category]:not(" + selector + ")").addClass("hidden");
         $("form#report_inspect_form " + selector).removeClass("hidden");
+        // And update the associated priority list
+        var priorities = $("form#report_inspect_form " + selector).data('priorities');
+        var $select = $('#problem_priority'),
+            curr_pri = $select.val();
+        $select.find('option:gt(0)').remove();
+        $.each(priorities.split('&'), function(i, kv) {
+            kv = kv.split('=', 2);
+            $select.append($('<option>', { value: kv[0], text: decodeURIComponent(kv[1]) }));
+        });
+        $select.val(curr_pri);
```

Comment:
If there are no priorities found, this sets the label of the control to 'undefined':
<img width="467" alt="screen shot 2016-12-15 at 15 20 43" src="https://cloud.githubusercontent.com/assets/4776/21229799/b3bce04c-c2da-11e6-9600-9c73e7b1b552.png">